.github/workflows/ubuntu_clean_meson_build.yml

```diff
@@ -29,7 +29,74 @@ jobs:
     - run: meson setup build/
       env:
         CC: gcc
-    - run: meson --buildtype=plain --prefix=/usr --sysconfdir=/etc --libdir=lib/x86_64-linux-gnu --bindir=lib/nntrainer/bin --includedir=include -Dinstall-app=true -Dreduce-tolerance=false -Denable-debug=true -Dml-api-support=enabled -Denable-nnstreamer-tensor-filter=enabled -Denable-nnstreamer-tensor-trainer=enabled -Denable-nnstreamer-backbone=true -Dcapi-ml-common-actual=capi-ml-common -Dcapi-ml-inference-actual=capi-ml-inference -Denable-capi=enabled build
+    - run: |
+        meson \
+          --buildtype=plain \
+          --prefix=/usr \
+          --sysconfdir=/etc \
+          --libdir=lib/x86_64-linux-gnu \
+          --bindir=lib/nntrainer/bin \
+          --includedir=include \
+          -Dinstall-app=true \
+          -Dreduce-tolerance=false \
+          -Denable-debug=true \
+          -Dml-api-support=enabled \
+          -Denable-nnstreamer-tensor-filter=enabled \
+          -Denable-nnstreamer-tensor-trainer=enabled \
+          -Denable-nnstreamer-backbone=true \
+          -Dcapi-ml-common-actual=capi-ml-common \
+          -Dcapi-ml-inference-actual=capi-ml-inference \
+          -Denable-capi=enabled \
+          build
     - run: ninja -C build
     - name: run ninja test
       run: cd ./build && ninja test
+
+  meson_test_fp16:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-latest ]
```

Comment:
I think tests on latest version of ubuntu is sufficient.

Reply:
Thank you for implementing CI.
However, since there are many devices that are still testing and running NNtrainer on 20.04, could you please use both 20.04 and 22.04?
I believe that it is good for stability to conduct explicit tests even if Ubuntu LTS changes to 24.04 in the future.

Reply:
@DonghakPark I changed os to 20.04 and 22.04. Thank you for comment.